context/app/api/vitessce_confs/constants.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+
+
+class Assays(Enum):
```

Comment:
What would the alternative be?  I am open to options but I don't like just having a series of unstructured strings since these can be grouped nicely into two categories.

Reply:
Two separate files then? I think that would provide all the structure you need. Sorry if I'm missing something.